in_toto/model_test.go

```diff
@@ -1529,13 +1529,16 @@ func TestDecodeProvenanceStatement(t *testing.T) {
     { "name": "curl-7.72.0.tar.gz",
       "digest": { "sha256": "d4d5899a3868fbb6ae1856c3e55a32ce35913de3956d1973caccd37bd0174fa2" }}
   ],
-  "predicateType": "https://slsa.dev/provenance/v0.1",
+  "predicateType": "https://slsa.dev/provenance/v0.2",
   "predicate": {
     "builder": { "id": "https://github.com/Attestations/GitHubHostedActions@v1" },
-    "recipe": {
-      "type": "https://github.com/Attestations/GitHubActionsWorkflow@v1",
-      "definedInMaterial": 0,
-      "entryPoint": "build.yaml:maketgz"
+	"buildType": "https://github.com/Attestations/GitHubActionsWorkflow@v1",
```

Comment:
nit: is the indentation a little iffy here?

Reply:
thanks, spaces vs tabs always get me 😅